notebooks/cuml_benchmarks.ipynb

```diff
@@ -0,0 +1,997 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Benchmark and Bounds Tests\n",
+    "\n",
+    "The purpose of this notebook is to benchmark all of the single GPU cuML algorithms against their skLearn counterparts, while also providing the ability to find and verify upper bounds.\n",
+    "\n",
+    "Each benchmark returns a Panda with the results, which can then be analyzed, manipulated, and stored to disk. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Notebook Credits\n",
+    "**Authorship**\n",
+    "Original Author: Corey Nolet <br />\n",
+    "Last Edit: Taurean Dyer, 9/25/2019<br />\n",
+    "\n",
+    "Last Edit: Corey Nolet, 10/04/2019\n",
+    "    \n",
+    "### Test System Specs\n",
+    "Test System Hardware: DGX-1 \n",
+    "Test System Software: Ubuntu 16.04  \n",
+    "RAPIDS Version: 0.10.0pre - Conda Install  \n",
+    "Driver: 410.48\n",
+    "CUDA: 10.0  \n",
+    "\n",
+    "### Known Working Systems\n",
+    "RAPIDS Versions: 0.10+"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.14.0a+852.gc5770a0.dirty\n"
+     ]
+    }
+   ],
+   "source": [
+    "import cuml\n",
+    "import pandas as pd\n",
+    "\n",
+    "from cuml.benchmark.runners import SpeedupComparisonRunner\n",
+    "from cuml.benchmark.algorithms import algorithm_by_name\n",
+    "\n",
+    "print(cuml.__version__)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "N_REPS = 3  # Number of times each test is repeated\n",
+    "\n",
+    "DATA_NEIGHBORHOODS = \"blobs\"\n",
+    "DATA_CLASSIFICATION = \"classification\"\n",
+    "DATA_REGRESSION = \"regression\"\n",
+    "DATA_NLP = \"20newsgroups\"\n",
+    "\n",
+    "INPUT_TYPE = \"numpy\"\n",
```

Comment:
Should we suppress the warning about wrong data shape? `warnings.filterwarnings('ignore', 'Expected column ')`? they kind of cover all the output

Reply:
Sure. Are you thinking we should do this in the notebook or in `cuml.benchmark`?

Reply:
For now I've added to the notebook.